egs/librispeech/ASR/beam_search.py

```diff
@@ -0,0 +1,1821 @@
+# Copyright    2021  Xiaomi Corp.        (authors: Fangjun Kuang
```

Comment:
Shall we put this file in some subfolder?

Reply:
Sry, just deleted it.